readme.md

```diff
@@ -31,35 +31,43 @@ TODO:
 
 ## Installation
 
-`python3.8` required.
```

Comment:
why did you delete this? it is worth to mention that python 3.8 is required. e.g it won't work with python3.7 and lower

Reply:
All installation methods currently listed in the readme won't allow you to install tg with an unsupported Python version...

It wasn't my intention to remove that, though... I forgot to add it back while changing the headers, sorry :disappointed: